lib/ansible/utils/module_docs_fragments/asa.py

```diff
@@ -39,6 +43,10 @@ class ModuleDocFragment(object):
     default: null
   provider:
     description:
+      - B(Deprecated)
+      - "Starting with Ansible 2.5 we recommend using C(connection: network_cli)."
+      - For more information please see the L(Network Guide, ../network/index.html).
```

Comment:
http://docs.ansible.com/ansible/devel/network/getting_started/network_differences.html#id3

Reply:
Great idea, much clearer page. Done
Though I wonder if http://docs.ansible.com/ansible/devel/network/getting_started/network_differences.html#multiple-communication-protocols need an example on it, what do you think?